src/parts/TypePart.php

```diff
@@ -59,19 +60,38 @@ private function parseType(Map $data) {
 		$this->collectionFormat = $data->get('collectionFormat');
 		$this->default = $data->get('default');
 		$this->maximum = $data->get('maximum');
-		$this->exclusiveMaximum = $data->has('exclusiveMaximum') && $data->get('exclusiveMaximum');
+		$this->exclusiveMaximum = $data->get('exclusiveMaximum');
```

Comment:
Yes but if you set ``false`` in your model, you don't know if the user already defined that value or not whereas with ``null`` :)